kolibri/plugins/facility/assets/src/views/DataPage/ImportInterface/index.vue

```diff
@@ -114,8 +120,8 @@
         message: 'Import a CSV file to create and update users',
         context: 'Additional information about importing external spreadsheets.\n',
       },
-      export: {
-        message: 'Export',
+      downloadCSV: {
+        message: 'Download CSV',
```

Comment:
**Download** breaks the connection with the wording in the surrounding strings, I suggest we go with something like `Export users (CSV)`.